src/components/form-access/FormAccess.tsx

```diff
@@ -34,7 +34,7 @@ export type FormAccessProps = FormProps & {
    * An override property if fine grained access has been setup for this form.
    * @type {boolean}
    */
-  fineGrainedAccess?: boolean;
+  fineGrainedAccess?: boolean | undefined;
```

Comment:
This field is already optional.
```suggestion
  fineGrainedAccess?: boolean;
```